code401Challenges/src/test/java/code401Challenges/linkedlist/LinkedListTest.java

```diff
@@ -0,0 +1,59 @@
+package code401Challenges.linkedlist;
+
+import org.junit.Test;
+import sun.awt.image.ImageWatched;
+
+import java.util.ArrayList;
```

Comment:
It looks like you're not using this import statement either! If you're not using an import, delete it!